File: one-unity/core/development/common/game-account/Runtime/Scripts/Model/CreateGuestResponse.cs
using System;
using Newtonsoft.Json;

namespace TPFive.Game.Account
{
    [Serializable]
    public class CreateGuestResponse
    {
        [JsonProperty("status")]
        public string Status { get; set; }

        [JsonProperty("message")]
        public Guest GuestInfo { get; set; }
    }
}

File: one-mobile/flutter_project/lib/xrauth/model/create_guest_response.dart
import 'dart:convert';
import 'package:tpfive/xrauth/model/guest.dart';

class CreateGuestResponse {
  String status;
  Guest guestInfo;

  CreateGuestResponse({
    required this.status,
    required this.guestInfo,
  });

  static CreateGuestResponse parse(String str) =>
      CreateGuestResponse.fromJson(json.decode(str));

  static String toJsonString(CreateGuestResponse data) =>
      json.encode(data.toJson());

  factory CreateGuestResponse.fromJson(Map<String, dynamic> json) =>
      CreateGuestResponse(
          status: json['status'], guestInfo: Guest.fromJson(json['message']));

  Map<String, dynamic> toJson() => {
        'status': status,
        'message': guestInfo.toJson(),
      };
}
